shared/middleware/net_sim_api.rb

```diff
@@ -45,6 +45,16 @@ def get_table(shard_id, table_name)
     get_table(shard_id, table_name).to_a.to_json
   end
 
+  # GET /v3/netsim/<shard-id>/<table-name>@<min_id>
+  #
+  # Returns all of the rows in the table with id >= min_id
+  #
+  get %r{/v3/netsim/([^/]+)/(\w+)@([0-9]+)$} do |shard_id, table_name, min_id|
```

Comment:
Looks like you didn't add a test for this route.  I know it's _almost_ just a pass-through to your other code under test, but it's nice to see the actual API proven.  Maybe omitted because you expect this to change soon anyway?